js/analyzer.js

```diff
@@ -303,16 +347,32 @@ Analyzer.prototype.imageAlttag = function(image){
 };
 
 /**
+ * checks if the alttag matches the keyword
+ * @param image
+ * @returns {boolean}
+ */
+Analyzer.prototype.imageAlttagKeyword = function(image){
+    var hasKeyword = false;
+    if(image !== null){
+        if(image[0].match(this.keywordRegex) !== null){
```

Comment:
Please combine this line with the line above to lower the complexity of this method.